src/main/scala/uk/gov/hmrc/sca/services/WrapperService.scala

```diff
@@ -65,10 +65,11 @@ class WrapperService @Inject()(
              backLinkUrl: String = "#",
              showSignOutInHeader: Boolean = false,
              scripts: Option[Html] = None,
-             showChildBenefitBanner: Boolean = false,
+             showChildBenefitBanner: Boolean = appConfig.showChildBenefitBanner,
              showUserResearchBanner: Boolean = false,
```

Comment:
can showUserResearchBanner be removed, and showHelpImproveBanner moved in its place instead? I think the user research banner was going to be the helpimprove banner but now it's not needed